README.md

```diff
@@ -97,7 +97,7 @@ You don't need to migrate database manually since migrations are executed every
 When you need to make ad-hoc connect to the database you can execute following command:
 
 ```
-docker-compose exec claim-store-api psql -U postgres -d claimstore
+docker exec -it cmc-integration-tests_shared-database_1 psql -U postgres -d claimstore
```

Comment:
The only problem with this is that we introduce dependency between repositories. If someone changed integration-tests to remove database or change service name it is hard to see that this README needs an update as well.

Reply:
This is what I do for ad-hoc connection:
```
docker-compose exec shared-database psql -U postgres -d claimstore
```
In the integration tests project

Although I have it configured in my intellij which is what I use 99% of the time, more if I ever need to connect to idam or pay etc

Reply:
Agreed, I have removed it from here and will update the README in integration-tests instead.